File: Python/fm-pipe.py
from subprocess import Popen, PIPE

demodulador = Popen(['python', 'fm-stdout.py'], stdout=PIPE)
fmedia = Popen(['python', 'log.py'],stdin=demodulador.stdout, stdout=PIPE)
# fmedia = Popen(['fmedia', '@stdin.wav', '--rate=44100','--channels=mono','--format=float32'],stdin=demodulador.stdout, stdout=PIPE)
# demodulador.stdout.close() # enable write error
out, err = fmedia.communicate()

File: Python/pruebas-fallidas/fm-pipe.js
const { spawn } = require('child_process');
const fs = require('fs');

const demodulador = spawn('python',['fm-stdout.py'])
const fmedia = spawn('fmedia',['@stdin.wav', '--rate=44100','--channels=mono','--format=float32'])

const file = fs.createWriteStream('muestras.log')
demodulador.stdout.pipe(file)

// child.stdout.on('data', (data) => {
//   console.log(`child stdout:\n${data}`);
// });